docs/install.md

```diff
@@ -0,0 +1,58 @@
+<!--
+SPDX-FileCopyrightText: 2021 Markus Sommer
+
+SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+# Installation Instructions
+
+## Manual
+
+- Install the *Go Programming language*, either through your package manager, or from [here](https://golang.org/dl/).
+- Clone the [source repository](https://github.com/dtn7/dtn7-go).
+- Check out the most recent release tag. Or don't and just build the `HEAD`, but we don't promise that it won't be broken.
+- (Optional) Run `GO111MODULE=on go test ./...` in the repository root to make sur we didn't screw up too badly.
```

Comment:
Nit: "make sure", not "sur"

Reply:
The `GO111MODULE` environment variable is a thing of the past. As the lowest supported Go version is 1.13, we can just ignore this one; compare [release notes](https://golang.org/doc/go1.13#modules).

Furthermore, for tests the race detector can be used, as we are doing within the CI. On the downside, this takes a lot more of your time and consumes more resources. For testing scenarios, one can also build `dtnd` with the `-race` flag to detect some faulty behavior not covered by the tests.

```suggestion
- (Optional) Run `go test ./...` in the repository root to make sure we didn't screw up too badly.
```